nifcloud/resources/dhcpoption/schema.go

```diff
@@ -0,0 +1,88 @@
+package dhcpoption
+
+import (
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/nifcloud/terraform-provider-nifcloud/nifcloud/validator"
+)
+
+const description = "Provides a dhcp options resource."
+
+// New returns the nifcloud_dhcp_option resource schema.
+func New() *schema.Resource {
+	return &schema.Resource{
+		Description: description,
+		Schema:      newSchema(),
+
+		CreateContext: create,
+		ReadContext:   read,
+		DeleteContext: delete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Default: schema.DefaultTimeout(5 * time.Minute),
+			Delete:  schema.DefaultTimeout(10 * time.Minute),
+		},
+	}
+}
+
+func newSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"default_router": {
+			Type:        schema.TypeString,
+			Description: "The IP address of default gateway.",
+			Optional:    true,
+			ForceNew:    true,
+			ValidateDiagFunc: validator.Any(
+				validator.IPAddress,
+			),
+		},
+		"domain_name": {
+			Type:        schema.TypeString,
+			Description: "The domain name used by the client in host name resolution.",
+			Optional:    true,
+			ForceNew:    true,
+		},
+		"domain_name_servers": {
+			Type:        schema.TypeList,
+			Description: "The IP address of the DNS server.",
```

Comment:
As with the document, you should write the list type in description.

```suggestion
			Description: "The IP address list of the DNS server.",
```